rotkehlchen/tests/unit/decoders/test_convex.py

```diff
@@ -0,0 +1,1171 @@
+import pytest
+
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.accounting.structures.base import (
+    HistoryBaseEntry,
+    HistoryEventSubType,
+    HistoryEventType,
+)
+from rotkehlchen.assets.asset import EthereumToken
+from rotkehlchen.chain.ethereum.constants import ZERO_ADDRESS
+from rotkehlchen.chain.ethereum.contracts import EthereumContract
+from rotkehlchen.chain.ethereum.decoding import EVMTransactionDecoder
+from rotkehlchen.chain.ethereum.decoding.constants import CPT_GAS
+from rotkehlchen.chain.ethereum.modules.convex.constants import CONVEX_POOLS, CPT_CONVEX
+from rotkehlchen.chain.ethereum.modules.convex.decoder import BOOSTER
+from rotkehlchen.chain.ethereum.structures import EthereumTxReceipt, EthereumTxReceiptLog
+from rotkehlchen.chain.ethereum.types import string_to_ethereum_address
+from rotkehlchen.chain.ethereum.utils import multicall
+from rotkehlchen.constants import ONE, ZERO
+from rotkehlchen.constants.assets import A_CRV, A_CVX, A_ETH
+from rotkehlchen.constants.ethereum import EthereumConstants
+from rotkehlchen.db.ethtx import DBEthTx
+from rotkehlchen.fval import FVal
+from rotkehlchen.types import EthereumTransaction, Location, deserialize_evm_tx_hash
+from rotkehlchen.utils.hexbytes import hexstring_to_bytes
+from rotkehlchen.utils.misc import hex_or_bytes_to_address
+
+
+def test_convex_pools(ethereum_manager):
```

Comment:
add a docstring explaining what this test does

Reply:
:+1: